podcasts/New Search/Views/History/SearchHistoryCell.swift

```diff
@@ -76,7 +79,7 @@ struct SearchHistoryCell: View {
                         }
                         .allowsHitTesting(false)
                     } else if let searchTerm = entry.searchTerm {
-                        Image("custom_search")
+                        Image("search")
                             .frame(width: 48, height: 48)
```

Comment:
Should this be set to `.frame(width: 56, height: 56)`  as well?

Reply:
Done in 0386e3ba2e429abc5e34040b18e082b78652e58b